examples/ts-react-chat/src/routes/api.realtime-token.ts

```
@@ -0,0 +1,135 @@
+import { createFileRoute } from '@tanstack/react-router'
+import { realtimeToken } from '@tanstack/ai'
+import { openaiRealtimeToken } from '@tanstack/ai-openai'
+import { elevenlabsRealtimeToken } from '@tanstack/ai-elevenlabs'
+import {
+  getCurrentTimeToolDef,
+  getWeatherToolDef,
+  setReminderToolDef,
+  searchKnowledgeToolDef,
```

ESLint

[error] Member 'searchKnowledgeToolDef' of the import declaration should be sorted alphabetically.

(sort-imports)